man/systemd.unit.xml

```diff
@@ -1602,14 +1602,24 @@
           <term><varname>ConditionControlGroupController=</varname></term>
 
           <listitem><para>Check whether given cgroup controllers (e.g. <literal>cpu</literal>) are available
-          for use on the system.</para>
+          for use on the system or whether the legacy v1 cgroup or the modern v2 cgroup hierarchy is used.
+          </para>
 
           <para>Multiple controllers may be passed with a space separating them; in this case the condition
           will only pass if all listed controllers are available for use. Controllers unknown to systemd are
-          ignored. Valid controllers are <literal>cpu</literal>, <literal>cpuset</literal>,
-          <literal>io</literal>, <literal>memory</literal>, and <literal>pids</literal>. Even if available in
-          the kernel, a particular controller may not be available if it was disabled on the kernel command
-          line with <varname>cgroup_disable=controller</varname>.</para></listitem>
+          ignored. Valid controllers are <literal>cpu</literal>, <literal>cpuacct</literal>,
+          <literal>io</literal>, <literal>blkio</literal>, <literal>memory</literal>,
+          <literal>devices</literal>, and <literal>pids</literal>. Even if available in the kernel, a
```

Comment:
I do not think it is not necessary to restore `blkio` and `devices`.

Only relevant value for this PR is `v2`. We should restore only minimal values (`v2`, or `v1` and `v2` at most).

Reply:
I dropped v1 controller names now (cpuacct, blkio, devices).